sessions/css-basics/shapes/styles.css

```
@@ -0,0 +1,11 @@
+body {
+  background: lightcoral;
+}
+
+.shapes {
+  width: 75px;
+  height: 75px;
+  margin: 30px;
+  background: black;
+  box-shadow: 0 0 5px 0 black;
+}
```

I personally find the background-color of the body and the box-shadow of the boxes a bit distracting and too "noisy/loud". I would probably remove them to simplify the exercise a bit.
```suggestion
body {
  box-sizing: boder-box;
}

.shapes {
  width: 75px;
  height: 75px;
  margin: 30px;
  background-color: black;
}
```